MAUI/Maps/legend.md

```diff
@@ -11,6 +11,11 @@ documentation: ug
 
 Using a legend, you can provide clear information on the data plotted on the map.
 
+To learn more about our .NET MAUI Maps legend, you can check the below video.
```

Comment:
To learn more about the .NET MAUI Maps legend, you can check the following video.